src/main/java/net/dzikoysk/funnyguilds/data/database/element/SQLTable.java

```diff
@@ -52,18 +52,24 @@ public String getName() {
         return name;
     }
 
+    public String getNameGraveAccent() {
+        return "`" + name + "`";
+    }
+
     public ArrayList<SQLElement> getSqlElements() {
         return sqlElements;
     }
 
-    public boolean containKey(String key) {
-        for (SQLElement element : sqlElements) {
-            if (element.getKey().equalsIgnoreCase(key)) {
-                return true;
+    public int getIndexElement(String key) {
+        for (int index = 0; index < sqlElements.size(); index++) {
+            if (!sqlElements.get(index).getKey().equalsIgnoreCase(key)) {
+                continue;
             }
+
+            return index;
         }
 
-        return false;
+        return 0;
```

Comment:
Jak nie znajduje elementu, to lepiej zwracać -1, żeby nie dostać tego samego wyniku co dla 0 elementu w tablicy i nie pomylić ze sobą tych wyników.